packages/swirl-components/src/components/swirl-date-input/swirl-date-input.spec.tsx

```diff
@@ -1,16 +1,34 @@
 const maskSpy = jest.fn();
```

Comment:
Issue: The tests are technically green, but running them locally `yarn stencil:test:watch` produces some errors caused by the setTimeout usage in focus().